R/esqlabs-plot-export-configuration.R

```diff
@@ -1,14 +1,5 @@
 #' @title ExportConfiguration
 #' @description R6 class defining properties for saving a `ggplot` object
-#' @field name character defining the name of the file to be saved (without extension)
-#' @field path Path of the directory to save plot to: path and filename are
-#' combined to create the fully qualified file name. Defaults to the working directory.
-#' @field format character defining the format of the file to be saved
-#' @field width numeric values defining the width in `units` of the plot dimensions after saving
-#' @field height numeric values defining the height in `units` of the plot dimensions after saving.
```

Comment:
Please explicitly document `height` as this is the change as compared with the default implementation @Felixmil

Reply:
Reworked that part a bit.